R/DataPull.R

```diff
@@ -0,0 +1,57 @@
+.getPreferenceScoreDist <- function(connection = NULL, 
+                                    preferenceScoreDist = NULL,
+                                    targetId,
+                                    comparatorId,
+                                    outcomeId,
+                                    databaseId,
+                                    analysisId) {
+  if (!is.null(connection)) {
+  sql <-   "SELECT *
```

Comment:
Indentation? Please use ctrl+i in R-Studio to set consistent indentation everywhere.